magda-web-client/src/AppContainer.scss

```diff
@@ -1,71 +1,81 @@
-@import './variables';
+@import "./variables";
```

Comment:
In the future, formatting edits like these should really be done in separate PRs from meaningful code changes, for the sake of review sanity.

Reply:
[There was an attempt](https://github.com/TerriaJS/magda/commit/9bc2ec88c92039219eab2afa9db9c6a86a3d3fcd) :(